Vagrantfile

```diff
@@ -49,13 +49,17 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |box|
     box.memory = 1024
     box.cpus = 1
+    box.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
   end
 
+  config.vm.provision :shell, :inline => 'apt-get update'
```

Comment:
`default.rd` ran `include_recipe "git"` before `"apt"`, which was causing the problem (apt was hitting incorrect locations and yelling a bit/bunch).

I am very new to Vagrant, so I was unaware that simply swapping this order would resolve the issue.  Seems it does, so I will push that and remove this shell command.